content/blog/using-jina-embeddings-haystack/index.md

```diff
@@ -0,0 +1,198 @@
+- **Fully open source 💙** There are both small and large embedding models available, depending on your computing resources and requirements. To run the embedding models yourself, [check out this documentation on HuggingFace](https://huggingface.co/jinaai/jina-embeddings-v2-base-en).  Alternately, you can use Jina's fully managed embedding service to handle that for you, which we'll be doing for this demo.
+
+## Getting started using Jina Embeddings v2 with Haystack
+
+To use the integration you'll need a free Jina api key - get one [here](https://jina.ai/embeddings/). 
+
+You can use Jina Embedding models with two Haystack components: `JinaTextEmbedder` and `JinaDocumentEmbedder`.
+
+To create semantic embeddings for documents, use`JinaDocumentEmbedder` in your indexing pipeline. For generating embeddings for queries, use `JinaTextEmbedder`. 
+
+In the following code we'll demonstrate how to use both components. You can also [see the Haystack docs for some minimum viable code examples.](https://haystack.deepset.ai/integrations/jina)
+
+
+## Summarizing legal text with a Haystack RAG pipeline
+
+I'm not a lawyer, and neither are large language models. But LLMs are good at analyzing long, complex documents. So let's try using the Jina v2 embedding models for some legal summarization.
+
+In October 2023, I narrowly escaped jury duty. I had slight FOMO since the case sounded interesting (Google v. Sonos). Let's see how it turned out.
+
+To follow along with this demo, in addition to a Jina api key you'll also need a [Hugging Face access token](https://huggingface.co/docs/hub/security-tokens), since we'll use the [Mixtral 8x7b LLM](https://mistral.ai/news/mixtral-of-experts/) for question answering.
+
+
+First, let's install all the packages we'll need.
+
+```bash
+pip haystack-ai jina-haystack chroma-haystack pypdf
+```
+Then let's input our credentials. Or you can set them as environment variables instead if you're feeling fancy.
+
+```python
+import getpass
+
+jina_api_key  = getpass.getpass("JINA api key:")
+hf_token = getpass.getpass("Enter your HuggingFace api token:")
+```
+
+## Building the indexing pipeline
+
+Our indexing pipeline will preprocess the legal document, turn it into vectors, and store them. We'll use the [Chroma DocumentStore](https://docs.trychroma.com/getting-started) to store the vector embeddings, via the [Chroma Document Store Haystack integration](https://haystack.deepset.ai/integrations/chroma-documentstore).
+
+```python
+from chroma_haystack.document_store import ChromaDocumentStore
+document_store = ChromaDocumentStore()
+```
+
+At a high level, the `LinkContentFetcher` pulls this document from its URL. Then we convert it from a PDF into a Document object Haystack can understand.
```

Comment:
Linking to the documentation of LinkContentFetcher would be a nice idea here as well.